schedule/virt_autotest/sle16_guest_installation.yaml

```diff
@@ -2,15 +2,19 @@ name:          sle16-virt-guest-installation
 description:    >
   Virtualization guest installation tests including extended feature tests schedule with agama and IPXE
 schedule:
-  - installation/ipxe_install
-  - installation/agama_reboot
+  - '{{host_installation}}'
   - virt_autotest/login_console
   - virt_autotest/prepare_non_transactional_server
   - virt_autotest/unified_guest_installation
   - virt_autotest/set_config_as_glue
   - '{{uefi_guest_verification}}'
   - '{{sev-es_guest_verification}}'
 conditional_schedule:
+  host_installation:
+    SKIP_HOST_INSTALL:
```

Comment:
Is `SKIP_HOST_INSTALL` required to be set in all test suites? or no change is required?

Reply:
It needs to be set 0 if not skipping. 
See the change in default x86, in https://gitlab.suse.de/qa-testsuites/openqa-job-group-yaml/-/merge_requests/276/diffs. So no testsuite will need to set it.